docs/pages/push-notifications/using-fcm.md

```diff
@@ -17,17 +17,35 @@ Note that FCM is not currently available for Expo iOS apps.
 
 4. In your app.json, add an `android.googleServicesFile` field with the relative path to the `google-services.json` file you just downloaded. If you placed it in the root directory, this will probably look like
 
-```javascript
-{
-  ...
-  "android": {
-    "googleServicesFile": "./google-services.json",
+  ```javascript
+  {
     ...
+    "android": {
+      "googleServicesFile": "./google-services.json",
+      ...
+    }
   }
-}
-```
+  ```
 
-Finally, make a new build of your app by running `expo build:android`.
+5.  Confirm that the API key in the `google-services.json` file has the correct restrictions in the [Google Cloud Platform API Credentials console](https://console.cloud.google.com/apis/credentials). Firebase requires the API key to have access to both `Firebase Cloud Messaging API` and `Firebase Installations API` for push notifications to work correctly. The API key in the `google-services.json` file can be found under the `client.api_key.current_key` field, e.g.
+
+  ```javascript
+  {
+    ...
+    "client": [
+      {
+        "api_key": [
+          {
+            "current_key" "<your Google Cloud Platform API key>",
+          }
+        ],
+      }
+    ]
+  }
```

Comment:
I think since we already state it can be found under `client.api_key.current_key`, this might be a little extraneous

Reply:
I'd prefer to keep this in, unless you have major objections. I am personally a more visual reader/learner, and the snippets are helpful for me.